teamcity/resource_build_config.go

```diff
@@ -826,11 +826,19 @@ func expandBuildFeature(raw interface{}) (api.BuildFeature, error) {
 func expandBuildSteps(list interface{}) ([]api.Step, error) {
 	out := make([]api.Step, 0)
 	in := list.([]interface{})
+	names := make(map[string]struct{})
 	for _, i := range in {
 		s, err := expandBuildStep(i)
 		if err != nil {
 			return nil, err
 		}
+
+		if _, exist := names[s.GetName()]; exist {
```

Comment:
This change seems unrelated to cron trigger support.

Reply:
I'm working on fixing that, not sure why they got included.

Reply:
Done